lxd/instance/drivers/driver_qemu.go

```diff
@@ -2309,6 +2325,63 @@ func (d *qemu) deviceAttachBlockDevice(deviceName string, configCopy map[string]
 	return nil
 }
 
+func (d *qemu) unmountPath(deviceName string, rawConfig deviceConfig.Device) error {
+	// First, cleanly unmount the path inside the VM.
+	msg := map[string]any{
+		"action": "removed",
+		"name":   deviceName,
+		"config": rawConfig,
+	}
+
+	err := d.devlxdEventSend("device", msg)
```

Comment:
This actually seems like a problem now im looking at it because we are going to be sending a duplicated removed event, as its also sent once removal is finished (the correct time) in qemu.Update.

Perhaps we can send an `unmount` event?

Reply:
fixed